lua/neoclip/handlers.lua

```diff
@@ -91,11 +91,24 @@ end
 
 -- TODO can this be done without setting the register?
 M.paste = function(entry, op)
+    if op == "v" then
```

Comment:
[nit] should this be done inside the function passed to `temporary_reg_usage` below in order to call this function at two places? Eg
```lua
    temporary_reg_usage(entry, function(register_name)
        if op == 'v' then
            vim.api.nvim_feedkeys('gv"'..register_name..'p', "n", false)
        else
            vim.cmd(string.format('normal! "%s%s', register_name, op))
        end
    end)
```
also is `nvim_feedkeys` required here or could it also use `vim.cmd`, I guess the visual part maybe breaks this?